frontend/src/home/Home.tsx

```diff
@@ -132,16 +133,6 @@ const LogoutButton = styled.div`
   margin-right: 8px !important;
 `;
 
-const ColorButton = withStyles((theme: Theme) => ({
```

Comment:
I believe @andrewjleung was also working on similar button refactors. What we have here looks good, just @'ing to make him aware.

Reply:
@justineluo and I ended up leaving out button refactors, so this should be all fine 👍